0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_siege_ram.xml

```diff
@@ -2,9 +2,9 @@
 <Entity>
   <Identity>
     <History>
-        The battering ram was one of the most popular siege weapons in the ancient world, and was used to damage walls or make them collapse.
-        A large pole was hung by ropes on a horizontal beam. It was pulled back and released. The pole swung forward, hitting the wall at maximum speed. The people operating the ram were protected by a roofed shell. The whole construction rolled on wheels because it was too heavy to be carried.
-        Defenders tried to stop battering rams by throwing heavy rocks at them or trying to set them on fire.
+        Battering rams are one of, if not the single, most popular siege engines of all time. They came in all different sizes and shapes and were generally very easy to construct. The earliest versions were just groups of soldiers carrying long poles, often tree logs, and ramming them against the targets.
+        However, during classical antiquity, 0 A.D.'s timeframe, battering rams had already been developed further: The pole was usually hung by ropes or chains. When ramming, it was pulled back and released. The pole then swung forward, its head, often reinforced with iron or steel, hitting the target. Because the entire heavy pole's force was concentrated on the tip, even small rams could break through strong gates and crack meter-thick walls.
```

Comment:
```suggestion
        However, during classical antiquity, 0 A.D.'s time, battering rams had been developed further: The pole was usually hung from a frame by ropes or chains. For ramming, the pole was pulled back and released. The pole swung forward, and its head, often reinforced with iron or steel, hitting the target. Because all the heavy pole's force was concentrated at the tip, even small rams could break through strong gates or crack thick walls.
```